README.md

```diff
@@ -2,7 +2,7 @@
 
 ![logo](logo-enterprise-search.png)
 
-The home of Elastic Enterprise Connector Packages. Use connector packages to
+The home of Elastic Enterprise Connector Clients. Use connector clients to
```

Comment:
Should do a search of `Enterprise` in the whole repo and clean up accordingly.

Reply:
What's our stance on `Enterprise`?

Reply:
I'd vote to keep it unless we hear otherwise. cc: @serenachou

Reply:
My understanding is the new connector has nothing to do with Enterprise Search, therefore I feel it's inappropriate to see `Enterprise` in this repo.

Reply:
@wangch079 We're still under "Enterprise Search" section in Kibana so far, afaik
![image](https://user-images.githubusercontent.com/12238374/180014863-ba000bec-d459-4894-99fd-e420074ab775.png)

Reply:
Correct, the products are still part of Enterprise Search, and the decoupling that happens at a technical level is a different matter. Please keep the naming as it was assigned for Connectors 2.0 and we'll review it when the time comes.